less/index.less

```
@@ -57,3 +57,178 @@ html, body {
     height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+//Variables
+@background-color: lightgray;
+@font-family: Tahoma, Helvetica, serif;
+@height: 100%;
+@padding: 10px;
+@font-size: 4rem;
+@width: 45%;
+@flex-direction: column;
+@justify-content: center;
+@table-background-color: rgb(199, 196, 196);
+
+//Mixins
+.middle-left-right {
+	display: flex;
+	flex-direction: column;
+	width: 50%;
+}
+
+// Nested at-rule
+// @media (max-width: 500px) {
+// 		//background: @background-color;
+// 		//.middle-right;
+// 		//.middle-left-right;
+// 		flex-flow: wrap;
+// 		width: 100%;
+
+// 		flex-direction: column;
+// }
+
+.container {
+	font-size: 1.6rem;
+	width: 75%;
+	margin: 0 auto;
+	@media (max-width: 500px) {
+		width: 100%;
+	}
```

```suggestion
	.container {
	font-size: 1.6rem;
	width: 75%;
	margin: 0 auto;
	@media (max-width: 500px) {
		width: 100%;
	}
```
I would reverse these and have 100% on the container and less on the breakpoint